crates/matrix-sdk/src/event_cache/mod.rs

```diff
@@ -590,9 +590,9 @@ impl RoomEventCacheInner {
     /// storage, notifying observers.
     async fn replace_all_events_by(
         &self,
-        events: Vec<SyncTimelineEvent>,
+        sync_timeline_events: Vec<SyncTimelineEvent>,
         prev_batch: Option<String>,
-        ephemeral: Vec<Raw<AnySyncEphemeralRoomEvent>>,
+        sync_ephemeral_events: Vec<Raw<AnySyncEphemeralRoomEvent>>,
```

Comment:
No need for the `sync_` prefix for ephemeral events: as opposed to the `sync_timeline_events` which make a distinction between `TimelineEvent` and `SyncTimelineEvent` (two different types that exist in the Rust SDK), there's no `SyncEphemeralEvent` in the code base, and the fully qualified name with the type in it would be `any_sync_ephemeral_room_event` which we don't really care about, so let's keep it short :)

```suggestion
        ephemeral_events: Vec<Raw<AnySyncEphemeralRoomEvent>>,
```